packages/server/lib/browsers/chrome.coffee

```diff
@@ -216,8 +271,22 @@ module.exports = {
         ## by being the last one
         args.push("--user-data-dir=#{userDir}")
         args.push("--disk-cache-dir=#{cacheDir}")
+        debug("get random port %d for remote debugging", port)
+        args.push("--remote-debugging-port=#{port}")
 
         debug("launch in chrome: %s, %s", url, args)
 
-        utils.launch(browser, url, args)
+        # FIRST load the blank page
+        # first allows us to connect the remote interface,
+        # start video recording and then
+        # we will load the actual page
+        utils.launch(browser, null, args)
```

Comment:
The `null` URL makes Chrome load `google.com`, this should probably be `about:blank` instead, I think that was the original intention